### PR TITLE
Make my code cleaner and fix a bug

### PR DESCRIPTION
**Pull request by ManIsCat2**

### CODE REVIEW

File: src/saturn/imgui/saturn_imgui_machinima.cpp [hunk 1]
```diff
@@ -411,7 +410,7 @@ void imgui_machinima_quick_options() {
     ImGui::Checkbox("Fog", &enable_fog);
     imgui_bundled_tooltip("Toggles the fog, useful for near-clipping shots");
     ImGui::Checkbox("Rainbow DisplayLists", &rainbow);
-    imgui_bundled_tooltip("Sets the displaylists to be Rainbow.");
+    imgui_bundled_tooltip("Makes the displaylists Rainbow.");
```
Comment (fgetcat) [inline src/saturn/imgui/saturn_imgui_machinima.cpp:413]: can you make the "rainbow" not capitalized plz

Comment (ManIsCat2) [reply src/saturn/imgui/saturn_imgui_machinima.cpp:413]: > can you make the "rainbow" not capitalized plz

no

Comment (fgetcat) [reply src/saturn/imgui/saturn_imgui_machinima.cpp:413]: death

Comment (Deniiguess) [reply src/saturn/imgui/saturn_imgui_machinima.cpp:413]: why am i here

**Pull request merged**